install.sh

```diff
@@ -604,6 +604,11 @@ install_werf_system_dependencies() {
     return 0
   fi
 
+  VERSION_ID=$(lsb_release -r | cut -f2)
+  if [[ "$VERSION_ID" == "18.04" ]]; then
```

Comment:
I don't even see the check whether this is Ubuntu or not. And what about Debian? Or other deb-based distros? Non-deb distros?

Alright, thinking about it now I don't think your approach will work at all. Instead we should do this:
1. If git is not present, then install it through package manager.
1. Check git version, if it is too old, then proceed to the next step.
1. Determine whether this is deb-based system, so that the PPA can be added.
a. If deb-based, then install `add-apt-repository`, then add ppa and do apt update and apt install.
b. If not deb-based, then do it through the respective package managers.